lib/mixpanel-node.js

```diff
@@ -295,13 +291,7 @@ var create_client = function(token, config) {
             properties = {};
         }
 
-        // time is optional for `track` but must be less than 5 days old if set
-        if (properties.time) {
-            properties.time = ensure_timestamp(properties.time);
-            if (properties.time < Date.now() / 1000 - TRACK_AGE_LIMIT) {
```

Comment:
None of our other SDKs perform this check. If we want to keep it we'll need to determine if the provided timestamp is in seconds or milliseconds, but doesn't seem necessary to me.